dashboard/test/helpers/delete_accounts_helper_test.rb

```diff
@@ -1914,6 +1913,55 @@ def assert_bucket_hard_deletes_contents(bucket)
     end
   end
 
+  #
+  # Project tables and kvps i.e. datablock storage
+  #
+
+  # Table: dashboard.datablock_storage_tables, dashboard.datablock_storage_records
+  test "Datablock Storage: hard-deletes all of user's project tables" do
+    student = create :student
+    with_channel_for student do |project_id_a, _|
```

Comment:
I've been searching for how to do `with_channel_for` several times now, HERE IT IS